### DISCUSSION TITLE
Implementation of admin-api `createProject` is hidden from this repo

### DISCUSSION THREAD

**Discussion by newVincentFong** [Unleash/unleash#5229] [General]
Well, well, well. I'ma to give you that
I was trying to modify some code to see if I can unlock multi-project feature on an OSS version of Unleash
Turned to find out, there is no `createProject` in ProjectApi
Seems like there is another repo lilke `unleash-enterprise`, which holds the complete code of all
Nice move btw
Now I have to give up choosing this cute platform
Cause we don't use Feature Flag at a very large scale but still want some multi-project abilities

**Comment by Tymek** [comment]
Hi @newVincentFong! 

I'm sorry to see you disappointed by our FOSS project. Last year we opened [constraints](https://www.getunleash.io/blog/unleash-brings-powerful-constraints-feature-to-oss-users) and this year we made [segments](https://www.getunleash.io/blog/unleash-5-5-strategy-variants-open-source-segments-and-more) available for free. I also have good news! You don't _have to_ give up :) We have plans designed for very scale. Our [Pro offer](https://www.getunleash.io/pricing) starts at 5 seats and include 3 projects. You're also more then welcome to create your own implementation of projects.
